packages/stats/gbstats/bayesian/tests.py

```diff
@@ -174,17 +177,16 @@ def compute_result(self):
             self.stat_a.unadjusted_mean,
         )
 
-        post_prec = (
-            1 / data_variance
-            + int(self.prior_effect.proper) / self.prior_effect.variance
+        post_prec = 1 / data_variance + (
+            0 if not scaled_prior_effect.proper else 1 / scaled_prior_effect.variance
```

Comment:
nit: would be easier to read as 
```1 / scaled_prior_effect.variance if scaled_prior_effect.proper else 0```

Reply:
yes, much better. I was being hasty. thanks